docs/tutorials/ionq/getting_started.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>Huh, it's an interesting thought to check for minimum cirq version - we could add that to our other notebooks as well. </p><p>we are at 0.10, so instead of <code>int(version[1]) &gt;= 9</code> it's either <code>&gt;9</code>  or <code>&gt;=10</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>fixed</p>